packages/server/src/providers-auth/providers-auth.service.ts

```diff
@@ -0,0 +1,95 @@
+import { getRepository } from 'typeorm';
+import passport, { Profile } from 'passport';
+import { Strategy as FacebookStrategy } from 'passport-facebook';
+// import { OAuthStrategy as GoogleStrategy } from 'passport-google-oauth'
+
+import UserEnity from '../models/user.entity';
+
+import RegistrationDto from '../models/registration.dto';
+
+import { ProvidersIdDBFieldName, ProvidersServiceName } from './providers-auth.interfaces'
+
+export default class ProvidersAuthService {
+  private loginUser;
+  private registerUser;
+  private repostory = getRepository(UserEnity);
```

Comment:
I would suggest naming as userRepository for consistency

Reply:
Is there typo in repostory name?

Reply:
Done